### PR TITLE
Ive added a bunch of the new librarys from the offical vertx project

### PR DESCRIPTION
**Pull request by johnfg10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,6 +342,7 @@ For Vert.x version 2 check [this page](./vert-x2.md).
 - [User Group](https://groups.google.com/forum/?fromgroups#!forum/vertx) - Discuss all user issues related to *using* Vert.x.
 - [Developer Group](https://groups.google.com/forum/?fromgroups#!forum/vertx-dev) - A group for Vert.x core *developers* and *contributors*.
 - [IRC channel](https://webchat.freenode.net/?channels=#vertx) - This is our day-to-day office: #vertx on freenode.net.
+- [Glitter chat](https://gitter.im/eclipse-vertx/vertx-users) general chat for Vert.x releated chat.
```
Comment (michel-kraemer) [inline README.md]: It's Gitter and not Glitter :)

Comment (vietj) [inline README.md]: typo : Gitter
can you also say that the IRC channel is replaced by the Gitter chat ?

Comment (vietj) [reply README.md]: actually you can remove the IRC channel

File: README.md [hunk 2]
```diff
@@ -270,6 +279,8 @@ For Vert.x version 2 check [this page](./vert-x2.md).
 
 ## Development Tools
 
+* [Vert.x shell](https://github.com/vert-x3/vertx-shell)  <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - allows interaction with Vert.x from the command line.
+* [Vert.x health check](https://github.com/vert-x3/vertx-health-check) - allows remote health checking in Vert.x projects.
```
Comment (michel-kraemer) [inline README.md]: Please change "allows" to "Allows for" (uppercase + for)

File: README.md [hunk 3]
```diff
@@ -176,7 +183,9 @@ For Vert.x version 2 check [this page](./vert-x2.md).
 ## Reactive
 
 * [Reactive Streams](https://github.com/vert-x3/vertx-reactive-streams) <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - Vert.x Reactive Streams.
-* [Reactive Extensions](https://github.com/vert-x3/vertx-rx) <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - Vert.x Reactive Extensions.
+* [Vert.x Rx](https://github.com/vert-x3/vertx-rx) <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - Vert.x Reactive Extensions.
+* [Vert.x Sync](https://github.com/vert-x3/vertx-sync) <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - Vert.x fiber support.
+* [Kotlin coroutines](https://github.com/vert-x3/vertx-lang-kotlin/tree/master/vertx-lang-kotlin-coroutines) <img src="https://rawgit.com/vert-x3/vertx-awesome/d93d327/vertx-favicon.svg" alt="(stack)" title="Vert.x Stack" height="16px"> - Vert.X support for kotlin coroutines.
```
Comment (michel-kraemer) [inline README.md]: Please change "Vert.X" to "Vert.x"

**Comment by michel-kraemer** [comment]
Thanks for the effort of updating the list so thoroughly!

I have a few comments. Otherwise the PR looks OK for me.

Someone else should review it too - just to be on the safe side.

**Comment by johnfg10** [comment]
fixed the issues mentioned above

**Pull request merged**